nck/entrypoint/__init__.py

```diff
@@ -0,0 +1,17 @@
+# GNU Lesser General Public License v3.0 only
```

Comment:
Are we sure we need this file?

Reply:
Yes, if we don't put `__init__.py` files in directories, it causes import errors when running tests with `nosetests`. I tried to remove them when doing the refacto last week, and I had to put them back because of that.

Reply:
The content or the file itself? For the content, I paste the content of another __init__.py file that's why

Reply:
The content of the file is the license, we have to put it everywhere. I think Benoit was talking of the file itself :)

Reply:
@benoitgoujon was talking about the file itself. But as @gabrielleberanger said, the init file is visibly mandatory for nosetests

Reply:
Yes without this file, I think the module cannot be used in other python files

Reply:
hm maybe we should change the way we run our tests then. Because `__init__.py` files were very Python 2. Now, we are not supposed to need them, especially if they are blank.